tests/config/config-import.spec.ts

```diff
@@ -206,12 +315,12 @@ describe("Config import", () => {
         mockAxiosGet("https://myTeam.celonis.cloud/package-manager/api/spaces", [space]);
         mockAxiosPut("https://myTeam.celonis.cloud/package-manager/api/packages/node-id/move/space-id", {});
         mockAxiosPost(assignVariablesUrl, {});
+        mockAxiosGet("https://myTeam.celonis.cloud/package-manager/api/packages", [node]);
 
         await new ConfigCommand().batchImportPackages("./export_file.zip", true);
 
         const expectedFileName = testTransport.logMessages[0].message.split(LOG_MESSAGE)[1];
         expect(mockWriteFileSync).toHaveBeenCalledWith(path.resolve(process.cwd(), expectedFileName), JSON.stringify(importResponse), {encoding: "utf-8"});
 
-        expect(mockedPostRequestBodyByUrl.get(assignVariablesUrl)).toEqual(JSON.stringify([variableAssignment]));
-    })
-})
\ No newline at end of file
+        expect(mockedPostRequestBodyByUrl.get(assignVariablesUrl)).toEqual(JSON.stringify([variableAssignment]));    })
```

Comment:
I think we should write assertions on the `axios.post` spy instead

Reply:
Assertions on `mockedPostRequestBodyByUrl` are the same since they only exist when the mock implementation is called ([ref](https://github.com/celonis/content-cli/blob/c5ca8e034d75753c63946b9556fec5abe5e48078/tests/utls/http-requests-mock.ts#L38)).

Reply:
Sure, we can consider in the future if we want to improve this, since there are existing tests with this approach, I'm fine with leaving it this way